### PR TITLE
Clean up English `og.document` translations.

### PR DESCRIPTION
**Pull request by lukasgraf**
Clean up English `og.document` translations.

For overall comments on the process, please see PR #6806 

Jira: https://4teamwork.atlassian.net/browse/CA-883

### CODE REVIEW

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po [hunk 1]
```diff
@@ -309,46 +309,49 @@ msgstr "Document rendition suitable for archival"
...
-msgstr "Live Search: search for the dossier"
+msgstr "Live Search: Search for target dossier"
 
 #. German translation: Datum des Dokuments
 #: ./opengever/document/behaviors/metadata.py
 msgid "help_document_date"
-msgstr "help_document_date"
+msgstr "Document date"
 
 #. German translation: Datei, die zu einem Dossier hinzugefügt wird
```
Comment (njohner) [inline opengever/document/locales/en/LC_MESSAGES/opengever.document.po:337]: Wow now I feel like the German translation is pretty weird here

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po [hunk 2]
```diff
@@ -739,9 +748,10 @@ msgstr "Sequence Number"
 
 #. German translation: Dokumentdatum
 #. Default: "from"
+#. MARKER: double check
```
Comment (njohner) [inline opengever/document/locales/en/LC_MESSAGES/opengever.document.po:752]: 🤣

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po [hunk 3]
```diff
@@ -527,12 +527,12 @@ msgstr "Destination"
...
 #: ./opengever/document/behaviors/metadata.py
 msgid "label_digitally_available"
-msgstr "Digital Available"
+msgstr "Has digital file"
 
 #. German translation: In Bearbeitung
 #: ./opengever/document/browser/report.py
 msgid "label_document_checked_out_by"
-msgstr "label_document_checked_out_by"
+msgstr "Checked out by"
```
Comment (njohner) [inline opengever/document/locales/en/LC_MESSAGES/opengever.document.po:535]: This one also seems to have a unsuitable label. You should probably stick to something close to the German:
`Being edited`
Well ok, that's not great, but I don't find a good translation 😓

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po [hunk 4]
```diff
@@ -390,7 +390,7 @@ msgstr "Changed by"
 #: ./opengever/document/behaviors/metadata.py
 #: ./opengever/document/browser/overview.py
 msgid "label_archival_file"
-msgstr "Archival File"
+msgstr "Archival file"
```
Comment (njohner) [inline opengever/document/locales/en/LC_MESSAGES/opengever.document.po:393]: I think at some other place you used `Archive file`

**Review by njohner — APPROVED** [review]
LGTM, just a few comments...

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po [hunk 5]
```diff
@@ -256,6 +256,7 @@ msgstr "Either the title or the file is required."
 
 #. German translation: Archivdatei
 #. Default: "Archive file"
+#. MARKER: double check
 #: ./opengever/document/behaviors/metadata.py
 msgid "fieldset_archive_file"
 msgstr "Archive file"
```
[the remark this answers is not in the document]
Comment (lukasgraf) [reply opengever/document/locales/en/LC_MESSAGES/opengever.document.po:262]: I like that one. I also had something with `Archival` in mind, but wasn't quite sure enough if that's really a (substantial enough) improval. But I'll see if I can make `File for archival` happen.

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po (hunk 3 above)
Comment (lukasgraf) [reply opengever/document/locales/en/LC_MESSAGES/opengever.document.po:535]: I'm quite certain that column in the report gets filled out with an actual person's name (gets transormed with `readable_author`). It's not boolean, whether or not it's being edited, it's a _who_. So again the German one is weird I believe 😉

File: opengever/document/locales/en/LC_MESSAGES/opengever.document.po (hunk 4 above)
Comment (lukasgraf) [reply opengever/document/locales/en/LC_MESSAGES/opengever.document.po:393]: Indeed. I'll attempt to change both to `File for archival` as you suggested, I like that one.

**Pull request merged**